src/scripting/lua_ui.cc

```diff
@@ -367,6 +367,7 @@ const MethodType<LuaDropdown> LuaDropdown::Methods[] = {
    METHOD(LuaDropdown, open),
    METHOD(LuaDropdown, highlight_item),
    METHOD(LuaDropdown, select),
+   METHOD(LuaDropdown, get_no_of_items),
```

Comment:
Remove this. It's available as a property and we don't need to add a method too.